### PR TITLE
SEO optimization  tasks

### PR DESCRIPTION
**Pull request by christianAcctGlobal**
#### How should this be manually tested?

[Workspace](https://seotask2--powerplanet.myvtex.com/admin/cms/store)

#### Screenshots or example usage
image

#### Type of changes

| ✔️  | Type of Change                                                                            |
| --- | ----------------------------------------------------------------------------------------- |
| \_  | Bug fix                                |
| \_  | New feature                        |
| ✔️  | Breaking change  |
| \_  | Technical improvements  |

#### Related to / Depends on
[Store](https://github.com/vtex-apps/store/pull/543)

#### Notes
Some settings may take time to be reflected in the store

### CODE REVIEW

File: messages/bg.json [hunk 1]
```diff
@@ -558,5 +558,11 @@
   "admin/store.twitterUsername.title": "Потребителско име за Twitter",
   "admin/store.twitterUsername.description": "Потребителското име, което фирмата ви използва в Twitter, например: VTEXcommerce.",
   "admin/store.twitterCard.title": "Twitter Card",
-  "admin/store.twitterCard.description": "Моля, изберете вида Twitter Card"
+  "admin/store.twitterCard.description": "Моля, изберете вида Twitter Card",
+  "admin/store.advancedSettings.enablePageNumberTitle.title": "Активирайте номера на страницата в мета заглавието",
+  "admin/store.advancedSettings.enablePageNumberTitle.description": "Добавете #{номер} в края на заглавието, за да избегнете проблема с дублирането на заглавието",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.title": "Каноничен url без url параметри",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "Премахнете параметъра на картата от каноничния URL адрес. Може да причини проблеми, когато отидете на страница с избрани аспекти. Тествайте внимателно, след като го включите",
```
Comment (camilavcoutinho) [inline messages/bg.json:565]:
```suggestion
  "admin/store.advancedSettings.canonicalWithoutUrlParams.title": "Редовен URL адрес без URL параметри",
  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "Премахнете параметъра на картата от редовния URL адрес. Може да причини проблеми, когато отидете на страница с избрани аспекти. Тествайте внимателно, след като го включите",
```

File: messages/es.json [hunk 2]
```diff
@@ -561,5 +561,11 @@
   "admin/store.twitterUsername.title": "Nombre de usuario de Twitter",
   "admin/store.twitterUsername.description": "El nombre de usuario que su empresa utiliza en Twitter, por ejemplo VTEXcommerce.",
   "admin/store.twitterCard.title": "Twitter Card",
-  "admin/store.twitterCard.description": "Por favor, elija el tipo de Twitter Card"
+  "admin/store.twitterCard.description": "Por favor, elija el tipo de Twitter Card",
+  "admin/store.advancedSettings.enablePageNumberTitle.title": "Habilitar el número de página en el meta title",
```
Comment (camilavcoutinho) [inline messages/es.json:565]:
```suggestion
  "admin/store.twitterCard.description": "Por favor, elija el tipo de Twitter Card",
  "admin/store.advancedSettings.enablePageNumberTitle.title": "Activar el número de página en el metatítulo",
```

File: messages/es.json [hunk 3]
```diff
@@ -561,5 +561,11 @@
...
+  "admin/store.advancedSettings.enablePageNumberTitle.description": "Agregar un #{Número} al final del título para evitar el problema de la duplicación del título",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.title": "URL canónica sin parámetros de URL",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "Elimina el parámetro de mapa de la URL canónica. Puede causar problemas al acceder a una página con facetas seleccionadas. Pruebe con precaución después de activar",
```
Comment (camilavcoutinho) [inline messages/es.json:568]:
```suggestion
  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "Elimina el parámetro de mapa de la URL canónica. Puede causar problemas al acceder a una página con facetas seleccionadas. Prueba con precaución después de la activación.",
```

File: messages/fr.json [hunk 4]
```diff
@@ -558,5 +558,11 @@
   "admin/store.twitterUsername.title": "Nom d'utilisateur Twitter",
   "admin/store.twitterUsername.description": "Le nom d'utilisateur que votre entreprise utilise sur Twitter, par exemple : VTEXcommerce.",
   "admin/store.twitterCard.title": "Carte Twitter",
-  "admin/store.twitterCard.description": "Veuillez choisir le type de la Twitter Card"
+  "admin/store.twitterCard.description": "Veuillez choisir le type de la Twitter Card",
+  "admin/store.advancedSettings.enablePageNumberTitle.title": "Activer le numéro de page dans le meta title",
```
Comment (camilavcoutinho) [inline messages/fr.json:562]:
```suggestion
  "admin/store.advancedSettings.enablePageNumberTitle.title": "Activer le numéro de page dans la balise meta title",
```

File: messages/fr.json [hunk 5]
```diff
@@ -558,5 +558,11 @@
...
+  "admin/store.advancedSettings.enablePageNumberTitle.description": "Ajoutez un #{Nombre} à la fin du titre pour éviter le problème de duplication de titre",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.title": "URL canonique sans paramètres d'URL",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "Supprimer le paramètre de carte de l'URL canonique. Peut causer des problèmes lors de l'accès à une page avec des facettes sélectionnées. Testez avec prudence après l'avoir allumé",
```
Comment (camilavcoutinho) [inline messages/fr.json:565]:
```suggestion
  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "Supprimer le paramètre de carte de l’URL canonique. Peut causer des problèmes lors de l’accès à une page avec des facettes sélectionnées. À tester avec précaution après l’avoir activé",
```

File: messages/fr.json [hunk 6]
```diff
@@ -558,5 +558,11 @@
...
+  "admin/store.advancedSettings.removeStoreNameTitle.title": "Supprimer le nom du magasin du titre",
+  "admin/store.advancedSettings.removeStoreNameTitle.description": "Supprimer le nom du magasin à la fin des méta-titres des produits et des catégories"
```
Comment (camilavcoutinho) [inline messages/fr.json:567]:
```suggestion
  "admin/store.advancedSettings.removeStoreNameTitle.description": "Supprimer le nom du magasin à la fin des balises meta title des produits et des catégories"
```

File: messages/it.json [hunk 7]
```diff
@@ -561,5 +561,11 @@
   "admin/store.twitterUsername.title": "Nome utente su Twitter",
   "admin/store.twitterUsername.description": "Il nome utente che la tua azienda utilizza su Twitter, ad esempio: VTEXcommerce.",
   "admin/store.twitterCard.title": "Twitter Card",
-  "admin/store.twitterCard.description": "Si prega di scegliere il tipo di Twitter Card"
+  "admin/store.twitterCard.description": "Si prega di scegliere il tipo di Twitter Card",
+  "admin/store.advancedSettings.enablePageNumberTitle.title": "Abilita il numero di pagina nel meta title",
+  "admin/store.advancedSettings.enablePageNumberTitle.description": "Aggiungi un #{Numero} alla fine del titolo per evitare il problema della duplicazione del titolo",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.title": "URL canonico senza parametri URL",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "Rimuovi il parametro della mappa dall'URL canonico. Può causare problemi quando si accede a una pagina con i facet selezionati. Testare con cautela dopo averlo acceso",
```
Comment (camilavcoutinho) [inline messages/it.json:568]:
```suggestion
  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "Rimuovi il parametro della mappa dall'URL canonico. Può causare problemi quando si accede a una pagina con i facet selezionati. Testalo con cautela dopo averlo abilitato.",
```

File: messages/it.json [hunk 8]
```diff
@@ -561,5 +561,11 @@
...
+  "admin/store.advancedSettings.removeStoreNameTitle.title": "Rimuovi il nome del negozio dal titolo",
+  "admin/store.advancedSettings.removeStoreNameTitle.description": "Rimuovi il nome del negozio alla fine dei meta titoli del prodotto e della categoria"
```
Comment (camilavcoutinho) [inline messages/it.json:570]:
```suggestion
  "admin/store.advancedSettings.removeStoreNameTitle.description": "Rimuovi il nome del negozio alla fine dei meta title di Prodotto e Categoria"
```

File: messages/pt.json [hunk 9]
```diff
@@ -558,5 +558,11 @@
   "admin/store.twitterUsername.title": "Nome de usuário no Twitter",
   "admin/store.twitterUsername.description": "O nome de usuário que sua empresa usa no Twitter, por exemplo: VTEXcommerce.",
   "admin/store.twitterCard.title": "Twitter Card",
-  "admin/store.twitterCard.description": "Por favor, escolha o tipo de Twitter Card"
+  "admin/store.twitterCard.description": "Por favor, escolha o tipo de Twitter Card",
+  "admin/store.advancedSettings.enablePageNumberTitle.title": "Habilitar o número da página no meta título",
+  "admin/store.advancedSettings.enablePageNumberTitle.description": "Adicione um # {Número} ao final do título para evitar o problema de duplicação do título",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.title": "URL canônico sem parâmetros de url",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "Remova o parâmetro do mapa do URL canônico. Pode causar problemas ao acessar uma página com facetas selecionadas. Teste com cuidado depois de ligá-lo",
```
Comment (camilavcoutinho) [inline messages/pt.json:565]:
```suggestion
  "admin/store.advancedSettings.canonicalWithoutUrlParams.title": "URL canônico sem parâmetros de URL",
  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "Remova o parâmetro do mapa do URL canônico. Pode causar problemas ao acessar uma página com facetas selecionadas. Teste com cuidado depois de ativá-lo",
```

File: messages/pt.json [hunk 10]
```diff
@@ -558,5 +558,11 @@
...
+  "admin/store.advancedSettings.removeStoreNameTitle.title": "Remover o nome da loja do título",
+  "admin/store.advancedSettings.removeStoreNameTitle.description": "Remova o nome da loja no final dos metatítulos de produto e categoria"
```
Comment (camilavcoutinho) [inline messages/pt.json:567]:
```suggestion
  "admin/store.advancedSettings.removeStoreNameTitle.description": "Remova o nome da loja no final dos meta títulos Produto e Categoria"
```

File: messages/ro.json [hunk 11]
```diff
@@ -558,5 +558,11 @@
   "admin/store.twitterUsername.title": "Nume de utilizator Twitter",
   "admin/store.twitterUsername.description": "Numele de utilizator pe care îl folosește business-ul tău pe Twitter, de exemplu: VTEXcommerce.",
   "admin/store.twitterCard.title": "Twitter Card",
-  "admin/store.twitterCard.description": "Te rugăm să alegi tipul pentru Cardul Twitter"
+  "admin/store.twitterCard.description": "Te rugăm să alegi tipul pentru Cardul Twitter",
+  "admin/store.advancedSettings.enablePageNumberTitle.title": "Activați titlul numărului de pagină",
+  "admin/store.advancedSettings.enablePageNumberTitle.description": "Adăugați un #{Număr} la sfârșitul titlului pentru a evita problema de duplicare a titlului",
```
Comment (camilavcoutinho) [inline messages/ro.json:563]:
```suggestion
  "admin/store.advancedSettings.enablePageNumberTitle.title": "Activează numărul de pagină în titlul meta",
  "admin/store.advancedSettings.enablePageNumberTitle.description": "Adaugă un #{Număr} la sfârșitul titlului pentru a evita problema de duplicare a titlului",
```

File: messages/ro.json [hunk 12]
```diff
@@ -558,5 +558,11 @@
...
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.title": "Adresă URL canonică fără parametri URL",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "Eliminați parametrul hărții din adresa URL canonică. Poate cauza probleme atunci când accesați o pagină cu fațete selectate. Testați cu precauție după ce îl porniți",
+  "admin/store.advancedSettings.removeStoreNameTitle.title": "Eliminați numele magazinului din titlu",
+  "admin/store.advancedSettings.removeStoreNameTitle.description": "Eliminați numele magazinului la sfârșitul metatitlurilor de produse și categorii"
```
Comment (camilavcoutinho) [inline messages/ro.json:567]:
```suggestion
  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "Elimină parametrul hărții din adresa URL canonică. Este posibil să cauzeze probleme atunci când accesezi o pagină cu fațete selectate. Testează cu precauție după ce îl pornești",
  "admin/store.advancedSettings.removeStoreNameTitle.title": "Elimină numele magazinului din titlu",
  "admin/store.advancedSettings.removeStoreNameTitle.description": "Elimină numele magazinului de la finalul metatitlurilor de produse și categorii"
```

File: messages/ja.json [hunk 13]
```diff
@@ -558,5 +558,11 @@
   "admin/store.twitterUsername.title": "Twitter ユーザー名",
   "admin/store.twitterUsername.description": "あなたのビジネスが Twitter で使うユーザー名、例: VTEXcommerce.",
   "admin/store.twitterCard.title": "Twitter カード",
-  "admin/store.twitterCard.description": "Twitter カードの種類を選んでください"
+  "admin/store.twitterCard.description": "Twitter カードの種類を選んでください",
+  "admin/store.advancedSettings.enablePageNumberTitle.title": "メタタイトルでページ番号を有効にする",
+  "admin/store.advancedSettings.enablePageNumberTitle.description": "タイトルの重複の問題を回避するために、タイトルの最後に＃{番号}を追加します",
```
Comment (camilavcoutinho) [inline messages/ja.json:563]:
```suggestion
  "admin/store.advancedSettings.enablePageNumberTitle.description": "重複の問題を回避するために、タイトルの最後に#{番号} を追加します",
```

File: messages/ja.json [hunk 14]
```diff
@@ -558,5 +558,11 @@
...
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.title": "URLパラメータのない正規URL",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "正規URLからマップパラメータを削除します。ファセットが選択されているページに移動すると、問題が発生する可能性があります。電源を入れた後は注意してテストしてください",
```
Comment (camilavcoutinho) [inline messages/ja.json:565]:
```suggestion
  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "正規URLからマップパラメータを削除します。ファセットが選択されているページに移動すると、問題が発生する可能性があります。有効にしてからは注意してテストしてください",
```

File: messages/ko.json [hunk 15]
```diff
@@ -558,5 +558,11 @@
...
   "admin/store.twitterUsername.description": "귀하의 비즈니스가 Twitter에서 사용하는 사용자 이름. 예: VTEXcommerce.",
   "admin/store.twitterCard.title": "Twitter 카드",
-  "admin/store.twitterCard.description": "Twitter 카드의 종류를 선택하세요."
+  "admin/store.twitterCard.description": "Twitter 카드의 종류를 선택하세요.",
+  "admin/store.advancedSettings.enablePageNumberTitle.title": "메타 제목에서 페이지 번호 활성화",
+  "admin/store.advancedSettings.enablePageNumberTitle.description": "제목 중복 문제를 방지하려면 제목 끝에 #{숫자}를 추가하세요.",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.title": "URL 매개변수가 없는 표준 URL",
+  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "표준 URL에서 지도 매개변수를 제거합니다. 패싯이 선택된 페이지로 이동할 때 문제가 발생할 수 있습니다. 전원을 켠 후 주의해서 테스트하십시오.",
+  "admin/store.advancedSettings.removeStoreNameTitle.title": "제목에서 상점 이름 제거",
+  "admin/store.advancedSettings.removeStoreNameTitle.description": "제품 및 카테고리 메타 제목 끝에 있는 상점 이름 제거"
```
Comment (camilavcoutinho) [inline messages/ko.json:567]:
```suggestion
  "admin/store.advancedSettings.canonicalWithoutUrlParams.description": "표준 URL에서 지도 매개변수를 제거합니다. 패싯이 선택된 페이지로 이동할 때 문제가 발생할 수 있습니다. 설정 후 주의해서 테스트하십시오.",
  "admin/store.advancedSettings.removeStoreNameTitle.title": "제목에서 스토어 이름 제거",
  "admin/store.advancedSettings.removeStoreNameTitle.description": "제품 및 카테고리 메타 제목 끝에 있는 스토어 이름 제거"
```

**Review by camilavcoutinho — CHANGES_REQUESTED** [review]

File: messages/es.json (hunk 2 above)
Comment (christianAcctGlobal) [reply messages/es.json:565]: done

**Review by camilavcoutinho — APPROVED** [review]

**Review by sheilagomes — APPROVED** [review]
LGTM

**Pull request merged**